token/program/src/processor.rs

```diff
@@ -4216,6 +4374,17 @@ mod tests {
                 ],
             )
         );
+        assert_eq!(
+            Err(TokenError::NotRentExempt.into()),
+            do_process_instruction(
+                initialize_multisig2(&program_id, &multisig_key, &[&signer_keys[0]], 1).unwrap(),
+                vec![
+                    &mut multisig_account,
+                    &mut rent_sysvar,
```

Comment:
I hope this isn't needed!  It may make sense to have a succeeding test for initializing the multisig with `initialize_multisig2`

Reply:
Yeah good point.  My main goal here was just to validate the `initialize_multisig2` plumbing was correct and I just grabbed an existing test kinda at random.  But a positive test case is certainly better.  Done